src/network/networkprotocol.cpp

```diff
@@ -61,6 +61,7 @@
 		[scheduled bump for 5.10.0]
 	PROTOCOL VERSION 47
 		Add particle blend mode "clip"
+		Add artificial light color, beta_r0, vignette, specular intensity, foliage translucency and cdl parameters to Lighting packets
```

Comment:
Needs an update for 5.12.0

Reply:
Oh, I think I forgot to save the file before commiting :(
But I do already have it changed on my side so it will be fixed with the next commit.